dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -3519,19 +3519,6 @@ protected void checkLogsForErrorsAndWarnings(List<String> failuresRegExps, Strin
 
                                                      "Config13TCKServer", //com.ibm.ws.microprofile.config.1.3_fat_tck
 
-                                                     "mpGraphQL10.basicQuery", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.defaultvalue", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.graphQLInterface", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.iface", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.ignore", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.inputFields", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.jarInWar", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.outputFields", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.rolesAuth", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.types", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.ui", //com.ibm.ws.microprofile.graphql.1.0_fat
-                                                     "mpGraphQL10.voidQuery", //com.ibm.ws.microprofile.graphql.1.0_fat
-
```

Comment:
I was curious as to why some of the updated server.xmls weren't listed here.

It looks like `.basicMutation` and `.metrics` include `mpMetrics` which I assume fixes the version of `mpConfig` used and the test using `.deprecated` is disabled.

Reply:
Probably that the other tests did not repeat with the same feature sets ... but better to be consistent in case that changes.